Content.Shared/Movement/Pulling/Systems/PullingSystem.cs

```diff
@@ -452,11 +457,11 @@ public bool TogglePull(EntityUid pullerUid, PullerComponent puller)
             joint.MinLength = 0f;
             joint.Stiffness = 1f;
 
-            _physics.SetFixedRotation(pullableUid, pullableComp.FixedRotationOnPull, body: pullablePhysics);
+            // check if we actually need to set the FixedRotation to false to save a dirty call
+            if (pullablePhysics.FixedRotation && !pullableComp.FixedRotationOnPull)
+                _physics.SetFixedRotation(pullableUid, false, body: pullablePhysics);
         }
```

Comment:
Same here, copy-pasting code + comments isn't the solution.

Reply:
I... I didn't copy-paste anything though?